### ISSUE TITLE
Training just the adapter

### ISSUE THREAD

**Issue by DavenH** [yolomeus/adapter_bert#1]
Hi, and thanks for the repo. 

Since Adapter modules are designed specifically for fine-tuning pretrained models, I'm a bit confused about the structure of the project. The code seems that it defines the (untrained) BERT model with adapter layers interposed in certain places, where I would have expected code for inserting adapter layers into the graph of the pretrained model. 

Perhaps this repo is only for demonstrating where to put the adapter layers? If so, totally fine, and I can make a code contribution of the following if necessary -- but there's a few other things needed to get up and running with Adapters:
 
1) Loading a pretrained model's variables 
2) Freezing / stop-gradient them
3) Insert the adapter layers after the _attention_ and _output_ variables
4) Setting some good hyperparameters for adapter training

Does that sound right?
